_posts/2018/2018-05-13-a-job-should-know-whether-to-run-itself.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "A job should know whether to run itself"
+description: 'A little self.knowledge goes a long way.'
+layout: article
+category: ruby
+image:
+  base: '2018/a-job-should-know-whether-to-run-itself'
+  alt: 'A queue'
+  source: 'https://unsplash.com/photos/Nzb4LBsctyQ'
+  credit: 'Hal Gatewood'
+---
+
+Like other `ActiveWhatever` objects in Rails, a job also comes with a selection of callbacks that are run at different points in it's lifecycle.
+
+A list of the available callbacks are in the [Activejob Rails Guide](http://guides.rubyonrails.org/active_job_basics.html#callbacks).
+
+We can use these callbacks to save a lot of conditional logic for jobs that might not always need to be run.
```

Comment:
Are you saying we save ourselves time writing conditional logic for these jobs...or we can save writing conditional logic for the jobs that don't always need to be run...

I'm pretty sure you're saying the former, but for those unfamiliar with this the latter could be read.